src/SelectTeamChannelPicker.ts

```diff
@@ -0,0 +1 @@
+export * from "./controls/SelectTeamChannelPicker";
```

Comment:
Would it be better to rename the controls to `TeamChannelPicker`? In that case the name will be aligned with other pickers.